pkg/db/db.go

```diff
@@ -4183,3 +4179,87 @@ func (h *DBHandler) RunCustomMigrationEnvironments(ctx context.Context, getAllEn
 		return nil
 	})
 }
+
+type OverviewCacheRow struct {
+	EslId     EslId
+	Timestamp time.Time
+	Blob      string
+}
+
+func (h *DBHandler) ReadLatestOverviewCache(ctx context.Context, transaction *sql.Tx) (*OverviewCacheRow, error) {
+	if h == nil {
+		return nil, fmt.Errorf("readLatestOverviewCache: DBHandler is nil")
+	}
+	if transaction == nil {
+		return nil, fmt.Errorf("readLatestOverviewCache: no transaction provided")
+	}
+
+	span, _ := tracer.StartSpanFromContext(ctx, "readLatestOverviewCache")
```

Comment:
```suggestion
	span, ctx := tracer.StartSpanFromContext(ctx, "readLatestOverviewCache")
```
the returned context should be used.
Also please move all spans to the beginning of the function.